crypto/ui/ui_openssl.c

```diff
@@ -101,6 +101,12 @@
 
 # endif
 
+#if defined(OPENSSL_SYS_VXWORKS)
+#undef TERMIOS
+#undef TERMIO
+#undef SGTTY
+#endif
```

Comment:
Same thing here...  or rather, `#if` and `#endif` only need one space, all the `#undef` within need two spaces